google/cloud/forseti/services/server_config/service.py

```diff
@@ -26,6 +29,48 @@
 LOGGER = logger.get_logger(__name__)
 
 
+def _run(client):
+    """Runs Forseti.
+
+    Args:
+        client (ClientComposition): Forseti client object.
+
+    Returns:
+        ServerRunReply: the returned proto message.
+    """
+    today = datetime.now()
+    model_name = today.strftime('run_model_%Y%m%d_%H%M')
```

Comment:
Can you move these 2 lines to be closer to where they are actually being used?  i.e. where the model is created?

Reply:
Done